PROPERTIES.md

```diff
@@ -67,6 +71,43 @@ This file lists all the currently implemented Echidna property tests for ERC20,
 | ERC20-ALLOWANCE-001 | [test_ERC20_setAndIncreaseAllowance](https://github.com/crytic/properties/blob/main/contracts/ERC20/internal/properties/ERC20IncreaseAllowanceProperties.sol#L17) | Allowance should be updated correctly when `increaseAllowance` is called. |
 | ERC20-ALLOWANCE-002 | [test_ERC20_setAndDecreaseAllowance](https://github.com/crytic/properties/blob/main/contracts/ERC20/internal/properties/ERC20IncreaseAllowanceProperties.sol#L28) | Allowance should be updated correctly when `decreaseAllowance` is called. |
 
+# ERC721
+
+### Basic properties for standard functions
+
+| ID |  Name | Invariant tested |
+|---|---|---|
+| ERC721-BASE-001 | [test_ERC20_balanceOfZeroAddressMustRevert](https://github.com/crytic/properties/blob/main/contracts/ERC721/internal/properties/ERC721BasicProperties.sol#L13) | Calling `balanceOf` for the zero address should revert. |
```

Comment:
The link text seems refer to ERC20 instead of ERC721

Reply:
Updated